screens/SwipePage.jsx

```diff
@@ -233,12 +234,16 @@ const SwipePage = ({ setFavourites }) => {
     setLastTime(mySec);
   };
 
-  const handleAddToFavorite = (card) => {
-    //console.log("Added to favourite");
-    setFavourites((currCards) => [card, ...currCards]);
-    //console.log(card);
-    handleSwipeOnPress(1);
-    setTapCount(0);
+  const handleAddToFavorite = async (card) => {
+    try {
+      setFavourites((currCards) => [card, ...currCards]);
+      handleSwipeOnPress(1);
+      setTapCount(0)
+      const res = await postFavouritesByUserId(user, card.clothes_id);
+      console.log(res);;
+    } catch (err) {
+      console.log(err);
```

Comment:
we want to remove a new card from the favourites array once error appears here

Reply:
added now